org/lateralgm/components/ActionList.java

```diff
@@ -150,7 +146,7 @@ public void save()
 	 * @return The frame opened or <code>null</code> if no
 	 * frame was opened.
 	 */
-	public static MDIFrame openActionFrame(MDIFrame parent, Action a)
+	public MDIFrame openActionFrame(MDIFrame parent, Action a)
```

Comment:
Why did you make this nonstatic? It doesn't reference any instance variables, nor invoke any code that does, so passing it a this object has no effect....